packages/common/src/credentialDeploymentTransactions.ts

```diff
@@ -177,3 +182,35 @@ export function getAccountAddress(credId: string): AccountAddress {
     );
     return accountAddress;
 }
+
+export type CredentialInputV1 = {
+    ipInfo: IpInfo;
+    globalContext: CryptographicParameters;
+    arsInfos: Record<string, ArInfo>;
+    idObject: IdentityObjectV1;
+    revealedAttributes: AttributeKey[];
+    seed: string;
+    net: 'Testnet' | 'Mainnet';
```

Comment:
I believe this definition was also added for the identity type, so now we should have a shared type.